static/build_html.sh

```diff
@@ -8,12 +8,13 @@ JOB_NAME=$2
 
 STABLE_BASE_PATH="https://d2l.ai"
 PREVIEW_BASE_PATH="http://preview.d2l.ai/$JOB_NAME"
+PREVIEW_BASE_PATH_RELEASE="${PREVIEW_BASE_PATH/release/master}"
```

Comment:
What is release/master?

Reply:
should be http://preview.d2l.ai/d2l-en/master, where d2l-en is extracted from JOB_NAME?

Reply:
This is bash syntax to replace release with master. Notice the curly braces.

Reply:
Maybe we only need REPO_NAME? https://github.com/d2l-ai/d2l-en/blob/master/Jenkinsfile#L3

Reply:
There are multiple ways to implement this. But in any case, we cannot avoid JOB_NAME because it carries the PR number information which is required when PR previews are build and pdfs are correctly linked. So instead of passing one more argument, I just used the information from JOB_NAME.

Feel free to suggest an edit.

Reply:
Have you considered r0.17 branch and classic branch?

Reply:
The variable name `PREVIEW_BASE_PATH_RELEASE` is not readable. Why not simply:

```
alternate_base="${PREVIEW_BASE_PATH/release/master}"
```

Reply:
Otherwise LGTM

Reply:
> Have you considered r0.17 branch and classic branch?

Good catch, this logic is only for the release branch. It will not work on classic branch. Are we hosting r0.17 somewhere stably? r0.17 seems like just another branch now, right?

d2l.ai hosts the release. (STABLE)
classic.d2l.ai hosts the classic. (STABLE)
All other branches are like preview branches. (PREVIEW)

I can add a specific logic for classic like L14 and also give it separate variables.

Reply:
But in any case this is differnt for classic branch because we will never merge those. When sending a PR for classic branch i can add it there if we want this feature to be backported to the classic branch.

Reply:
Since r0.17 is in maintenance mode, let's only implement this change on actively-developed master/release

Reply:
Ok in that case this logic should be fine.

> The variable name PREVIEW_BASE_PATH_RELEASE is not readable. Why not simply:

I was just trying to be consistent with the naming. alternate_base was supposed to be calculated inside the if else but I'm fine with your suggested change if that looks better to you.